views/partials/_example.njk

```diff
@@ -0,0 +1,37 @@
+{% macro example(params) %}
+
+{% set examplePath = "src/"+params.group+"/"+params.item+"/"+params.example+".md.njk" %}
```

Comment:
Would prefer spaces around operators throughout. We should probably agree and document some code standards for nunjucks?